packages/dd-trace/src/llmobs/plugins/openai.js

```diff
@@ -159,6 +165,16 @@ class OpenAiLLMObsPlugin extends LLMObsPlugin {
   _tagChatCompletion (span, inputs, response, error) {
     const { messages, model, ...parameters } = inputs
 
+    const metadata = Object.entries(parameters).reduce((obj, [key, value]) => {
```

Comment:
We're moving this here to ensure input metadata is recorded even in case of error right?